client/state/ui/test/reducer.js

```diff
@@ -78,38 +78,38 @@ describe( 'reducer', () => {
 	} );
 
 	describe( '#section()', () => {
-		it( 'should not persist data because this is not implemented', () => {
+		it( 'should persist data', () => {
 			const state = section( 'hello', {
 				type: SERIALIZE
 			} );
 
-			expect( state ).to.eql( false );
+			expect( state ).to.eql( 'hello' );
```

Comment:
> Might make more sense to change this to a boolean.

`section` can be a string ('design', 'reader' etc) or `false`, so I think the string is ok, or am I missing something?